### DISCUSSION TITLE
MetaMaps

### DISCUSSION THREAD

**Discussion by LightLodges** [omigroup/omigroup#142] [R&D]
Weaving together threads from across our channels:

We are working on a collective OS endeavor to map interoperability across our wide field of fields.

I registered a domain for us to use here as a way to begin developing this open resource that can be hosted on git pages and populated in the coming weeks together. To start I'd love help organizing the PM dashboard at https://github.com/omigroup/omigroup/projects/3 for this project.

This is a huge effort and has engaged a few hundred people across your blogs along with Discord & Twitter primarily along with talks at a few conferences in recent weeks/months. There's a working HackMD notes doc at https://hackmd.io/w_sUTgcpRcSKdN6Torl-WA and I'll continue to drop our Twitter Spaces notes there as well for consolidation.

To summarize:
We'd like to build an open repo that maps interconnectedness factors and interop across metaverse teams/products/ecosystems as we explore how to do this work in the open together and build on each others' efforts. Ideally this 2D asset will be a 3D map with layers that can be toggled for a wide variety of potential use cases (educational directories, engineering, strategic development & design, interop policy & advocacy, collaboration etc).

We're in the process of sorting out what we need to be mapping for specifically at very granular levels and how to express this data. To do this work most effectively we'd love your insight on language, ways of expressing data, IA and UX especially as we build out the next stage of this endeavor. Questions/inquiries welcome and we'll be back on Twitter Spaces on Thursday evening for Open Q&A with weekly collab sessions TUESDAY AT NOON PST using Twitter Spaces (I'm hosting @amoration with colleagues @Open_Metaverse including Jesse @Mr Metaverse and Jin @DankVR). Next step for me is to begin mapping this to the Platform Design Toolkit in Miro to give us a 3D way to ideate and collaborate together.  That ideation board is weaving these threads together for R&D at https://miro.com/welcomeonboard/ZnI3R0k2RkF2UkJDWnpmdDU2clJVSkRGUVk1TFJrUERiQ2hFNXNwNjdYZHJwQVYwOUpkdUV5ajJpcGcxVlc4U3wzMDc0NDU3MzQ2ODI3NDU4ODkz?invite_link_id=773754257278

Now to the form - how do we best express this wide variety of data?

**Factors of Interoperability to Map include:**

    TYPE OF ORGANIZATION
        Business - privately held
        Business - publically owned
        Open Source org
        Nonprofit association
        Educational institution
        DAO or alt collective

    METAVERSE WORLD SUPPORT SYSTEMS
        Self-hosted
        Web-enabled
        Private/lockable
        Public/openable
        Capacity limit per space
        Standards

    VR CAPACITIES
        Immersive Web Accessibility
        Passthrough XR/OpenXR
        6DOF

    AR CAPACITIES

    SPATIAL WEB/PHYSICALITY
        Wearables or Haptics?
        Other Sensors Enabled
        Location-Based Interfaces
        IoT Enabled
        Real World Mapping
        Digital Twins Policy Links

    IDENTITY
        SSI
        Anonymity supported?
        Opt-out capacity
        Kids under 13?
        Teens 13-17?
        Data Privacy/Security Statement Links

    WEB3
        Supported ecosystems
        Multichain capacity?
        Wallets/plugins
        Link to Smart Contract or Whitepaper
        DAO supported?

    ASSETS (INCLUDE GLTF)
        Asset formats supported
        glTF portability
        Asset portability/downloadability?
        Inventory sharing?

    AVATARS (INCLUDE VRM)

    BROWSERS (WEBXR)

    METATRAVERSAL (PORTALS?)
    *

    MEDIA (STREAMING, RTMP)

    SOCIAL GRAPH

    CONCURRENCY/SCALABILITY

    ETHICAL ENGAGEMENT
        Links to Ethics/AI statements
        EULA links
        Community moderation?
        CyberXR Coalition iconography on equity, inclusion, diversity & accessibility for all

**Comment by LightLodges** [comment]
This conversation builds on the work of Jin at https://hackmd.io/@XR/maps and Doug at https://outofscope.bureauofbrightideas.com/a-map-of-the-metaverse/

**Comment by fire** [comment]
Through today's OMI meeting I described a MetaMaps proposal:

1. Airtable is used for information architecture
2. Forms are used to submit to Airtable
3. Airtable organization will be done as required
4. A person is able to ask the database for a mark representing the entity and a 3d position organized by factors
5. Ask for a 1000 entities of the above entry.
6. Given the 1000 entities run code to one click take the marks and the positions, place them into a 3D Blender scene, process and upload to Mozilla Hub
8. Fly through the MetaMap. 

## See also

https://support.airtable.com/hc/en-us/articles/360000342267-3D-space-app

## Open problems

1. Given factors calculate xyz position "force graph" of the list of marks
  2. https://www.npmjs.com/package/3d-force-graph
  3. Use each factor as way to weight the maps https://vasturiano.github.io/3d-force-graph/example/manipulate-link-force/
